jobs/pull-kubernetes-e2e-gce-cri.sh

```diff
@@ -50,7 +50,7 @@ export GINKGO_PARALLEL="y"
 # This list should match the list in kubernetes-e2e-gce.
 export GINKGO_TEST_ARGS='--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]'
 export FAIL_ON_GCP_RESOURCE_LEAK="false"
-export PROJECT="kubernetes-pr-cri-validation"
+export PROJECT="${PROJECT:-kubernetes-pr-cri-validation}"
```

Comment:
It was useful for testing out my branch in my own GCE project. I'd love to find a way to do this going forward if possible.